ddd-koans/Value Object Pt. 2 & Entity/Introduction/task.html

```diff
@@ -0,0 +1,95 @@
+<html>
+<h3>Feeling a little bit lost</h3>
+<p>After the first lesson about value object you can feel a little bit lost, if not that's good for you :)</p>
+<p>
+    In the first part of the <i>Value Object Pt. 2 & Entity</i> lesson you will use the value objects
+    (both known and unknown yet). They will be used to introduce the concepts from the music domain. After, the
+    first part, the new domain-driven design concept, the <b>Entity</b> will be explained and used by you.
+</p>
```

Comment:
are we going to keep this part?

Reply:
I can remove it, you haven't pointed that part to be a problem before.